### ISSUE TITLE
Round brackets in literary texts

### ISSUE THREAD

**Issue by HolgerEssler** [DCLP/dclpxsltbox#56]
Allow to encode round brackets which occasionally occur in literary texts in a syntactical function.

**Comment by paregorios** [comment]
I have the sense that we looked at an example of this, with photograph, but I cannot now find the note.

**Comment by HolgerEssler** [comment]
So far we encode the brackets directly in the xml. It is apparently not a problem of display, but of the Leiden+ converter, presumably due to the ambiguity with abbreviations (cf. abbreviated (Καρ(ανίδι)) vs. the parenthesis in this sentence).

**Comment by paregorios** [comment]
Yes, it's likely that the regular expression used in the xsugar grammar doesn't accommodate this particular variant, so we'll need to look at changing that.

**Comment by paregorios** [comment]
Example XML encoding by @andreabernini imported from #122:

P.Oxy. 70.4763 (TM 92143) col. I, 17-18
 * https://github.com/DCLP/idp.data/blob/dclp/DCLP/93/92143.xml#l95
 * https://github.com/DCLP/idp.data/blob/dclp/DCLP/93/92143.xml#l96

**Comment by paregorios** [comment]
@HolgerEssler and @andreabernini: 

I have a distinct memory of seeing this matter addressed either [Symbol Sanity for DCLP](https://docs.google.com/a/nyu.edu/spreadsheets/d/15Lnu9THFkYpCCUJIfnq3_DCJw72ivjUhHu3GLJQWY1k/edit#gid=0) or [LitPapText](https://docs.google.com/a/nyu.edu/spreadsheets/d/1F6_Du8DIjixRD4mnvfpbkcn3E0H2_SfBIztCO6Vl6gQ/edit#gid=1879498412) but I don't now find it in either place. Would you please see if you can find it and, if not, add appropriate entry or entries? 

And just to verify, you are happy with the display you're getting for these in http://dclp.github.io/dclpxsltbox/output/dclp/93/92143.html, i.e.:

```((round-bracket ))```

**Comment by andreabernini** [comment]
In LitPapText, ln. 11 there is an entry "Parentheses": in the XML and Leiden+ columns it is written          "(various types)". 
Such parentheses (as in the aforementioned P.Oxy. 70.4763 col. I, 17-18) occur in the ed.pr.: they are not in the text.

**Comment by paregorios** [comment]
Task:

 - [ ] sort out actually what we are looking for here.

**Comment by paregorios** [comment]
Elsewhere, @andreabernini has written "round brackets" where it's clear from context that "square brackets" were meant. Could that be the case here as well?

**Comment by paregorios** [comment]
I am still unclear as to what is meant by "round brackets" and whether these are to be understood as glyphs or marks or figures that actually appear on the papyri or whether they are sigla used by modern editors in editions. As we don't have clarity, I am moving this to the post-release milestone so we can concentrate on other priorities.

**Comment by rogerbagnall** [comment]
Round brackets, or parentheses as we say in English, appear in both roles. They are a Leiden convention editorial marking for abbreviations and the like, but they are also used with an entirely different function in papyri, both documentary and literary. To be sure, telling the difference between round and square in papyri is not always easy, but they do exist as ancient signs. Because the use of ( ) in Leiden+ is so specialized and requires the outer parentheses, one can't simply use that with the other significance to show what is on the papyrus. I don't know enough about their possible range of meaning, other than deletion, to say more about that. Perhaps Holger would like to.
